mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -47,13 +47,13 @@ class DeviceSerializer(serializers.Serializer):
         help_text="Current vehicle status.",
     )
     prev_event = serializers.ChoiceField(
-        source="latest_event.event_type",
+        source="latest_event[0].event_type",
```

Comment:
```suggestion
        source="latest_events[0].event_type",
```
No test failed? :stuck_out_tongue:

Reply:
:'(